bottom/client.py

```diff
@@ -102,8 +102,9 @@ def trigger(self, event: str, **kwargs: Any) -> None:
         async_event.set()
         async_event.clear()
 
-    async def wait(self, event: str) -> None:
+    async def wait(self, event: str) -> str:
```

Comment:
Please add a test for this.  Especially if preserving case is part of the contract, a test should enforce that a future code change doesn't start returning `event.upper()`.